### PR TITLE
#18 [PROGRESS] Ability to change icons on cards

### PR DESCRIPTION
**Pull request by singerxt**
https://github.com/x-team/unleash/issues/18

- [ ] remove debug code when PR ready
- [x] figure out how to "do not repeat yourself"
- [x] fix css issues.
- [x] working code

### CODE REVIEW

File: app/styles/pages/_templates.scss [hunk 1]
```diff
@@ -17,6 +17,91 @@ $template-alt-color: #ccc;
...
+    background-color: #f2f3f3;
+    color: #000000;
+    width: 100%;
+    height: 100%;
+    padding: 20px;
+    max-height: 100vh;
+    overflow-y: auto;
+    opacity: 0;
+    z-index: -100;
+    @include single-transition(opacity, 0.3s, linear);
```
Comment (theanxy) [inline app/styles/pages/_templates.scss]: Please insert @includes at the beginning of rulesets

File: app/styles/pages/_templates.scss [hunk 2]
```diff
@@ -17,6 +17,91 @@ $template-alt-color: #ccc;
...
+    h1 {
+      width: 100%;
+      text-align: center;
+      font-size: 40px;
+      border-bottom: 2px solid #fc6c7f;
+      margin-bottom: 10px;
+      padding-bottom: 10px;
+    }
+
+    .modal__icon {
```
Comment (theanxy) [inline app/styles/pages/_templates.scss]: You can use &__icon and &.current

**Comment by theanxy** [comment]
Looking good so far! Please make sure to squash commits and rebase on top of develop before sending the final PR.

**Comment by singerxt** [comment]
Of course. Thanks for feedback @theanxy

**Pull request closed without merging**